README.md

```diff
@@ -19,3 +19,9 @@ attributes from federated login.
 
 
 **template_s3_storage.yml**  Creates and assigns a cleanup policy to an Amzon S3 bucket for uploading files  
+
+
+
+To deploy a new stack: 
+   
+    aws cloudformation deploy --template-file /home/sg/IdeaProjects/NVA-infrastructure-test/deploy_nva_from_sar.yml --stack-name frontend-test --capabilities CAPABILITY_IAM CAPABILITY_AUTO_EXPAND --parameter-overrides AmazonCognitoDomain=https://nva-test.auth.eu-west-1.amazoncognito.com BackendDomain=api.test.nva.aws.unit.no CognitoIdentityPoolId=eu-west-1:c1187e43-cbb7-401f-8181-9c94d8e94b6a DomainCertificateArn=arn:aws:acm:us-east-1:812481234721:certificate/9319e9f9-3a66-43bb-9f9c-7fc1e6b4d3ab FrontendDomain=frontend.test.nva.aws.unit.no RedirectSignInUrl=https://frontend.test.nva.aws.unit.no RedirectSignOutUrl=https://frontend.test.nva.aws.unit.no/logout UserPoolAppClientId=77136duraj3vml8mi38j1hlhmv UserPoolId=eu-west-1_wsHsxCiT4 CognitoAuthorizerArn= arn:aws:cognito-idp:eu-west-1:812481234721:userpool/eu-west-1_wsHsxCiT4
```

Comment:
There seem to be details here specific to our current set up, use parameter names

Reply:
Yes, the command line is just a sample if needed. Less personalised, but still specifik for 'test'